packages/icons/web/v2/alelo_32.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="32" height="24" viewBox="0 0 32 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Optimized these a little 🙂 (the issue was that sometimes, icons are positioned between pixels on the Figma canvas, so when exporting them their size is off by 1px and they have a weird `clipPath` thing. When I do is copy them into a new Figma draft where I make sure they're positioned properly on the grid before exporting. I'll document all of this together with the icons contribution process)